cdap-docs/admin-manual/source/installation/mapr.rst

```diff
@@ -43,10 +43,47 @@ packages installed, and can be configured using the MapR `configure.sh
 .. --------------------
 .. include:: ../_includes/installation/hadoop-configuration.txt
 
+
+Create the "cdap" User
+----------------------
+
+.. highlight:: console
+   
+To prepare your cluster for CDAP, manually create a ``cdap`` user on all nodes of the
+cluster. As "...MapR uses each node's native operating system configuration to authenticate
+users and groups for access to the cluster...[MapR documentation]", make sure that the UID
+for the ``cdap`` user is the same on each node of the cluster::
+
+  $ useradd ... cdap
```

Comment:
wondering if the ``...`` may just lead to more questions.  I would think a full valid command, or something like "if using the useradd utility, the -u flag can be used..." would be better.  Looks like mapr just gives a full command:  http://doc.mapr.com/display/MapR/Preparing+Each+Node (common users)